<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MSelectPakage</class>
 <widget class="QDialog" name="MSelectPakage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>快递单号</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget_select">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>441</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_show">
    <attribute name="title">
     <string>快递一览</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_show">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>431</width>
       <height>461</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_show">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>429</width>
        <height>459</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>160</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_show"/>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_select">
    <attribute name="title">
     <string>已选中</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_select">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>431</width>
       <height>461</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_select">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>429</width>
        <height>459</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>160</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_select"/>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_commit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>590</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
